lib/rucio/tests/test_multi_vo.py

```diff
@@ -194,7 +194,13 @@ def test_rses_at_different_vos(self):
         rse_client.add_rse(tst)
         rse_client.add_rse(shr)
         add_rse(new, 'root', **self.new_vo)
-        add_rse(shr, 'root', **self.new_vo)
+        shr_id_new_original = add_rse(shr, 'root', **self.new_vo)  # Accurate rse_id for shared RSE at 'new'
+
+        shr_id_tst = get_rse_id(shr, **self.vo)
```

Comment:
Since there's a few different things being tested in this function now, it would benefit from some short comments saying what each part is testing.